Assets/HoloToolkit-Examples/UX/Scripts/OptimizeSceneforDeviceType.cs

```diff
@@ -2,35 +2,15 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 //
-using HoloToolkit.Unity;
-using System.Collections.Generic;
+
+using System;
 using UnityEngine;
-using UnityEngine.XR.WSA;
 
 namespace HoloToolkit.Unity.Examples
 {
+    [Obsolete("Use SceneContentAdjuster")]
     public class OptimizeSceneforDeviceType : MonoBehaviour
     {
         public GameObject containerObject;
```

Comment:
Why not delete public GameObject as well?

Reply:
To prevent broken links in the Inspector, to help make porting to the new script as easy as possible.